docs/release-notes/release-notes-0.2.0.md

```diff
@@ -0,0 +1,313 @@
+# DCRDEX v0.2.0 (beta)
+
+Apr 16, 2021
```

Comment:
While here, ought to link to downloads to resolve https://github.com/decred/dcrdex/issues/873 as well